configuration/examples/local/manifests/observatorium/tracing-servicemonitor-otel.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: otel-observatorium-xyz
+  namespace: observatorium
+spec:
+  endpoints:
+  - port: monitoring
```

Comment:
Do we enable metrics in the OTEL collector?

Reply:
yes, and the operator creates the monitoring service. So I only need to add the ServiceMonitor.